browser-test/src/admin/admin_program_visibility.test.ts

```diff
@@ -272,4 +273,47 @@ test.describe('Validate program visibility is correct for applicants and TIs', (
       'program-visibility-changes-all-ti-can-see-program',
     )
   })
+  test('create a program with disabled visibility, verify it is hidden from applicants and TIs', async () => {
+    const {page, tiDashboard, adminPrograms} = ctx
+
+    await enableFeatureFlag(page, 'disabled_visibility_condition_enabled')
+    await loginAsAdmin(page)
+
+    const programName = 'Disabled program'
+    const programDescription = 'Description'
+    await adminPrograms.addProgram(
+      programName,
+      programDescription,
+      'https://usa.gov',
+      ProgramVisibility.DISABLED,
+    )
+    await adminPrograms.publishAllDrafts()
+
+    // Login as applicant, verify program is hidden
+    await logout(page)
+    const applicantQuestions = new ApplicantQuestions(page)
```

Comment:
suggestion (blocking): pull this off of context at the top of the test like `const {page, tiDashboard, adminPrograms, applicantQuestions} = ctx`

Reply:
done!